site/design/displaying-data/index.jsx

```diff
@@ -41,6 +41,17 @@ export default (
         </figure>
       </div>
 
+      {/* Tree Grid */}
+      <div className={pf('site-side-by-side grid wrap p-vertical--x-large')}>
```

Comment:
question: Is it currently possible to disable the prefix through a sass variable?